### ISSUE TITLE
Can't Login to YTMusic

### ISSUE THREAD

**Issue by toltolminmin** [linsomniac/spotify_to_ytmusic#148]
When I use "ytmusicapi oauth" or "s2yt_ytoauth," I get a traceback that ends in "ValueError: relative path can't be expressed as a file URI"
![Screenshot 2024-12-17 192632](https://github.com/user-attachments/assets/463c15eb-e637-411f-9f1b-7398703658f7)

As for "python -m spotify2ytmusic ytoauth," it says there is "No module named spotify2ytmusic."

I don't understand the problem... because I'm tech-illiterate and have no experience in any of this. Please help!

**Comment by leotulipan** [comment]
got the same problem and for me https://github.com/sigma67/spotify_to_ytmusic worked now but i had to temporarily make the playlists i wanted to import public and the youtube browser login (request headers from chrome) is a bit technical but there are docs.

tldr: this other project worked for me

**Comment by toltolminmin** [comment]
Thank you!!
